### PR TITLE
Add endpoint to get account holder profile image

### PR DESCRIPTION
**Pull request by julesnuggy**
Add endpoint to get account holder profile image.

### CODE REVIEW

**Review by chengjoanna1729 — APPROVED** [review]

File: src/main/java/org/jarling/v2/api/AccountHolderResource.java [hunk 1]
```diff
@@ -14,4 +16,9 @@
      * Get name of the account holder
      */
     String getAccountHolderName() throws StarlingBankRequestException;
+
+    /**
+     * Get profile image of the account holder encoded as a Base64 string
```
Comment (kicferk1) [inline src/main/java/org/jarling/v2/api/AccountHolderResource.java]: Soo, is it a Base64 string(like the comment says), or a byte array(as the type and name suggest)? I imagine the latter.

**Review by kicferk1 — COMMENTED** [review]
Looks good apart from code comment looking out of date.

File: src/main/java/org/jarling/v2/api/AccountHolderResource.java (hunk 1 above)
Comment (julesnuggy) [reply src/main/java/org/jarling/v2/api/AccountHolderResource.java]: It is a byte array! Sorry, had changed the code but not this comment.

**Comment by StephenPenney** [comment]
Agreed looks good.

**Pull request merged**